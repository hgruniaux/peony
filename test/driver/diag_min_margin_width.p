//# RUN: peony -fdiagnostics-minimum-margin-width=0 %s
//# RUN: peony -fdiagnostics-minimum-margin-width=3 %s
//# RUN: peony -fdiagnostics-minimum-margin-width=8 %s

fn test() {
    5 + true;
    //~^ ERROR: cannot add 'i32' to 'bool'
}
