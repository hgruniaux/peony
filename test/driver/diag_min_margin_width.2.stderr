<source>:6:7: error: cannot add 'i32' to 'bool'
      6 |     5 + true;
        |     ~ ^ ~~~~
